lib/gobstones/batch_parser.rb

```diff
@@ -43,9 +43,17 @@ def parse_options(test)
       [
         struct(key: :show_initial_board, default: true),
         struct(key: :show_final_board, default: true),
-        struct(key: :check_head_position, default: false),
+        struct(key: :check_head_position, default: true),
```

Comment:
I am not sure of this change. Which is the default in xgobstones-runner?

Although It looks fine to me, I would like to avoid exercise incompatibilities.

Reply:
The default was `false`. I just changed it again to that value.